packages/core-ethereum/crates/core-ethereum-misc/src/chain.rs

```diff
@@ -692,112 +581,15 @@ pub mod tests {
         )
         .unwrap();
 
-        let params = super::convert_vrf_parameters(&acked_ticket.vrf_params);
-        let redeemable = super::convert_acknowledged_ticket(&acked_ticket).unwrap();
-
         let redeem_ticket_tx = Eip1559TransactionRequest::new()
             .from(H160::from_slice(&counterparty.public().to_address().to_bytes()))
             .to(hopr_channels.address())
-            .data(RedeemTicketCall { redeemable, params }.encode());
+            .data(chain.redeem_ticket(&acked_ticket).unwrap());
 
         println!(
             "{:?}",
             client.send_transaction(redeem_ticket_tx, None).await.unwrap().await
         );
-
-        // println!(
-        //     "{:?}",
-        //     hopr_channels.redeem_ticket(redeemable, params).send().await // .map_err(|e| {
-        //                                                                  //     println!("{:?}", e.decode_contract_revert::<HoprToken>());
-        //                                                                  // })
-        // );
-
-        // let on_chain_ticket_luck: Hash = hopr_channels
-        //     .get_ticket_luck(
-        //         acked_ticket.ticket.get_hash(&domain_separator).into(),
-        //         redeemable,
-        //         params,
-        //     )
-        //     .call()
-        //     .await
-        //     .unwrap()
-        //     .try_into()
-        //     .unwrap();
-
-        // println!("Ethereum ticket luck {}", on_chain_ticket_luck);
-        // println!(
-        //     "Offchain ticket luck {}",
-        //     hex::encode(acked_ticket.get_luck(&domain_separator).unwrap())
-        // );
-
-        // let hash: Hash = hopr_channels
-        //     .get_ticket_hash(redeemable)
-        //     .call()
-        //     .await
-        //     .unwrap()
-        //     .try_into()
-        //     .unwrap();
-
-        // println!("Ethereum hash {}", hash);
-        // println!("off-chain hash {}", ticket.get_hash(&domain_separator));
-
-        // client
-        //     .send_transaction(redeem_ticket_tx, None)
-        //     .await
-        //     .map_err(|e| e.to_string())
-        //     .unwrap();
-        // hopr_channels.redeem_ticket(redeemable, params)
-    }
-
-    #[async_std::test]
-    async fn redeem_ticket_workflow() {
```

Comment:
Is this not needed anymore?

Reply:
the control-flow logic has changed slightly. This is why the test does not make sense in that form